add image content-type header

darkrasid opened the pull request:
When I use image in Markdown, I find a bug.
If I link image in MD file from gitlab raw data, it would not show on IE. 
(eg. https://gitlab.com/user_id/repo_name/raw/master/image.png)

issue : https://gitlab.com/gitlab-org/gitlab-ce/issues/2376   

It is because of content-type of headers.
If the blob is not text, the content-type is always application/octet-stream. 
And IE cannot resolve it to image so it will be cracked. (I test it IE 10, 11) 
So I add code to change content-type if blob type is image. 

```
modified:   app/controllers/projects/raw_controller.rb
```

stanhu commented on CHANGELOG:
```diff
@@ -26,6 +26,8 @@ v 8.0.0 (unreleased)
   - Make all profiles public
   - Fixed login failure when extern_uid changes (Joel Koglin)
   - Don't notify users without access to the project when they are (accidentally) mentioned in a note.
+	- Fix bug which IE cannot resolve image problem in markdown when the image file is from raw data. 
```
Fix indentation.

stanhu commented:
Please rebase master. There is also a new spec in `raw_controller_spec.rb`; please add an image file to that and test this change.

Commit:
add Content-Type header of raw image file

Add Content-Type header to fix a bug which IE can't show image in
markdown when the image is from raw.

	modified:   CHANGELOG
	modified:   app/controllers/projects/raw_controller.rb
	modified:   spec/controllers/projects/raw_controller_spec.rb

darkrasid commented:
I rebased master and add test case for the raw_controller. 

If you want to check the situation of issue please access blew link with Internet Explorer 11, 10 or lower.
(https://gitlab.com/darkrasid/testtest/blob/master/test.md)

Thanks

stanhu commented:
There appears to be a merge conflict; possibly with CHANGELOG? Can you rebase once more and fixed the comment message?

darkrasid commented:
Hi. I rebase and fix comment. 
However there is a build error. I try to find cause and find some error of last gitlab build of master. (https://github.com/gitlabhq/gitlabhq/branches) 
Please check gitlab master branch.

stanhu commented:
Merging this because test failure was fixed in aa75e76a0a.
